common-services.yaml

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+#
+# SPDX-License-Identifier: MIT
+
+# This Docker Compose file defines common configuration for use in all other
+# Docker Compose files
+
+version: "3.7"
+
+services:
+  frontend:
+    build: ./client
+    restart: always
+    ports:
+      - 3005:3005
```

Comment:
This is actually something that could be left out of the template, as we do not need exposition between localhost and the container in use cases outside of local development

Reply:
Similarly to the other services